fluidreview/views.py

```diff
@@ -0,0 +1,19 @@
+"""Views for fluidreview"""
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from fluidreview.permissions import WebhookPermission
+from fluidreview.models import WebhookRequest
+
+
+class WebhookView(APIView):
+    """
+    Handle webhooks coming from FluidReview
+    """
+    permission_classes = (WebhookPermission,)
+    authentication_classes = ()
+
+    def post(self, request):
+        """Store webhook request for later processing"""
+        WebhookRequest.objects.create(body=request.body)
```

Comment:
Is it reasonable to expect that the webhook request body should always be in JSON format?  If so, maybe there should be validation for it.  But if there are situations where the data could legitimately be anything then this is fine as is.

Reply:
The webhook content is created using a plain text template. It should usually be JSON but it's possible someone may make a typo or syntax error when editing the template. In a future PR there will be an additional model to store data parsed from the JSON, but even then we should keep the raw text to aid in debugging problems